modules/client/examples/04-decoders/15-add-multisig-members.ts

```diff
@@ -0,0 +1,21 @@
+/* MARKDOWN
+### Decode Add Members Action (Multisig)
+*/
+import { MultisigClient, Context, ContextPlugin } from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+const multisigClient = new MultisigClient(contextPlugin);
+const data: Uint8Array = new Uint8Array([12, 56]);
+
+const members: string[] = multisigClient.decoding.addMembersAction(data);
```

Comment:
Same as before, if there's an extra param with the new amount of signers, the struct is going to change